frontend/public/components/app.jsx

```diff
@@ -147,6 +147,10 @@ class App extends React.PureComponent {
             <LazyRoute path="/cluster-health" exact loader={() => import('./cluster-health' /* webpackChunkName: "cluster-health" */).then(m => m.ClusterHealth)} />
             <LazyRoute path="/start-guide" exact loader={() => import('./start-guide' /* webpackChunkName: "start-guide" */).then(m => m.StartGuidePage)} />
 
+            <LazyRoute path="/marketplace/all-namespaces" exact loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */).then(m => m.MarketplacePage)} />
+            <LazyRoute path="/marketplace/ns/:ns" exact loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */).then(m => m.MarketplacePage)} />
```

Comment:
It looks like we'll only ever show marketplace items from the `marketplace` namespace. Is that right? Then we wouldn't need a route for each namespace.